crates/turbo-tasks/src/manager.rs

```diff
@@ -979,6 +985,22 @@ impl<B: Backend + 'static> TurboTasksApi for TurboTasks<B> {
     fn mark_own_task_as_finished(&self, task: TaskId) {
         self.backend.mark_own_task_as_finished(task, self);
     }
+
+    fn detached(
+        &self,
+        f: Pin<Box<dyn Future<Output = Result<()>> + Send + 'static>>,
+    ) -> Pin<Box<dyn Future<Output = Result<()>> + Send + 'static>> {
+        Box::pin(
+            TURBO_TASKS.scope(
+                turbo_tasks(),
+                CURRENT_TASK_ID.scope(
+                    CURRENT_TASK_ID.with(|id| *id),
+                    self.backend
+                        .execution_scope(CURRENT_TASK_ID.with(|id| *id), f),
```

Comment:
Nit: is it slow to double access? Could we `let current_task = CURRENT_TASK_ID.get()` and reuse it in both places?